model-metadata/model-metadata.md

```diff
@@ -4,35 +4,46 @@ This document describes the structure and content of a top-level Model Metadata
-| `model_type`         | [Model Type]\|string | **REQUIRED.** Identifier for the type of model. STRONGLY RECOMMENDED to use on of the standard [Model Type] values, but other values are allowed. |
-| `model_architecture` | string             | Identifies the model architecture used (e.g. RCNN, U-Net, etc.).                    |
-| `license`            | string             | **REQUIRED.** The model's license(s). Either a SPDX [License identifier], `various` if multiple licenses apply, or `proprietary` for all other cases. |
-| `authors`            | \[[Author]\]       | **REQUIRED.** List of names and contact information for the model author(s).                 |
-| `citation`           | [Citation]         | Citation information related to the model.                                          |
-| `training`           | [Model Training]   | **REQUIRED.** A description of the data and environment used to train the model.    |
-| `inputs`             | \[[Model Input]\]  | **REQUIRED.** A list of [Model Input] objects describing the names and types of model inputs.  |
-| `outputs`            | \[[Model Output]\] | **REQUIRED.** A list of [Model Output] objects describing the names and types of model outputs. |
-| `runtimes`           | \[[Runtime]\]      | A list of [Runtime] objects describing serialized or containerized versions of the model that can be used to generate inferences. |
-| `usage_recommendations` | [Usage Recommendations] | A description of the recommended conditions under which the model can be used. |
-
-### Algorithm Type
-
-This string describes the general type of machine learning algorithm employed by the model. It is STRONGLY RECOMMENDED 
-that you use one of the following values, but other values are allowed.
+| Field Name              | Type                     | Description                                                                                                                                                                                 |
+| ----------------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `model_id`              | string                   | **REQUIRED.** A unique string identifier for the model. This ID must be unique across the provider.                                                                                         |
+| `model_type`            | [Model Type]             | **REQUIRED.** Identifier for the type of model. STRONGLY RECOMMENDED to use on of the standard [Model Type] values, but other values are allowed.                                           |
+| `model_architecture`    | string                   | Identifies the model architecture used (e.g. RCNN, U-Net, etc.).                                                                                                                            |
```

Comment:
Is there a reason for leaving `model_architecture` outside of `model_type`?

Reply:
No, good catch. I was focused on the other two fields and just didn't move this one in. I'll update the PR with this in the `model_type` object.

Reply:
This was originally an optional field, but I'm wondering if it should be required as well. It seems like an easy thing for the publisher to provide...

Reply:
It should be required, imo.

Reply:
👍 . I also made the `learning_approach` and `prediction_type` required, since they were both required when they were in the top-level metadata and I failed to transfer that over originally.